src/main/java/com/sap/sgs/phosphor/fosstars/tool/github/SingleSecurityRatingCalculator.java

```diff
@@ -108,16 +103,14 @@ public SingleSecurityRatingCalculator calculateFor(List<GitHubProject> projects)
         new HasCompanySupport(github),
         new HasSecurityPolicy(github),
         new SecurityReviewForProject(github),
-        new UnpatchedVulnerabilities(github),
-        new VulnerabilitiesFromNvd(github),
+        new InfoAboutVulnerabilities(github),
         new IsApache(github),
         new IsEclipse(github),
         new LgtmDataProvider(github),
         new UsesSignedCommits(github),
-        new CompositeDataProvider<>(
-            new UsesOwaspDependencyCheck(github),
-            new UsesSnykDependencyCheck(github))
-            .stopWhenFilledOut(SCANS_FOR_VULNERABLE_DEPENDENCIES),
+        new ScansForVulnerableDependencies(github),
+
+        // TODO: currently interactive data provider have to be added to the end
```

Comment:
Is it possible to add an issue reference?

Reply:
Good point, I was going to open a ticket for that. We'll do. I'll mention this TODO in the ticket.